ibodata/lateral_profile.py

```diff
@@ -109,18 +109,48 @@ def normalize(self, dt, allow_cast=True):
         In case of corrupted data raises ValueError
         Translate y to bring y.min() to 0 (noise substraction) and then
         normalize to 1 over [-dt, +dt] area from the mid of the profile
+        if allow_cast is set to True, division not in place and casting may occur.
+        If division in place is not possible and allow_cast is False
+        an exception is raised.
         """
+        try:
```

Comment:
Explain in comments the reason of such `try` block. Sth like:

We will try now to divide the data by 1.0. If self.y is float, then nothing happens.
In case its has integers there are two cases:
 - conversion to float will happen blabla
 - integer will stay integers bla bla